MidgarSwift.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'MidgarSwift'
-  s.version          = '0.1.7'
+  s.version          = '0.1.8'
```

Comment:
I personally made it a minor bump for Kotlin and RN but that works for me as well.